DEV_QUICKSTART.md

```diff
@@ -0,0 +1,59 @@
+# Dev quickstart
+
+When you run `./gradlew runDev`, you will get a server running at `localhost:8080`.  If you then `cd` into the `client` directory, you can run:
+
+```
+nvm use   // get correct version of node & npm
+npm ci    // get dependencies
+npm start // start proxy server
+```
+
+and you will now have a browsersync proxy running at `localhost:3000` with hot reload for the react components and sass styles.
+
+> Note: as of now, in order to get hot reload of templates, you must run `./gradlew ide` and run `DevHotReload.java`.  We are trying to add hot reload into the regular `./gradlew runDev`, but for now you have to start the java IDE to get that.
```

Comment:
Why is this the case?

Reply:
From eclipse, you can hot reload the entire app - not just templates, but controllers too.  And that relies on the details of eclipse's incremental compiler.

If all we want is hotreload on templates (not controllers), then I'm pretty sure we can bake that into `runDev` without much trouble, I just haven't done it yet (I was confused, thought it was already done).